lib/generators/arclight/templates/arclight.scss

```diff
@@ -1,2 +1,3 @@
+$breadcrumb-divider: quote("»");
```

Comment:
When trying this out locally with a freshly generated .internal-test-app it seems that this variable is not getting applied when set in `arclight.scss` and the default Bootstrap `/` breadcrumb divider is displaying instead. It does work if I add `$breadcrumb-divider: quote("»");` at the top of `blacklight.scss`. Maybe there's something wrong with my local setup?

Reply:
Yes, I think so. `blacklight.scss` should be deleted via https://github.com/projectblacklight/arclight/pull/1202/files#diff-68e5bfaa20d28433265c9d3d44f4065954fde0d86389bbf2dce3a2ea61fa0730R46